### DISCUSSION TITLE
Switch to Vite

### DISCUSSION THREAD

**Discussion by martinszeltins** [vercel/next.js#68400] [Ideas]
### Goals

1. Align with all other major frameworks
2. Easier learning curve
3. Well known bundler

### Non-Goals

1.
2.
3.

### Background

All major frameworks except for React/Next have switched to Vite as their bundler so it makes sense for Next.js to adopt Vite as well.

### Proposal

N/A

**Comment by cwray-tech** [comment]
I very much ++++ this. Next is incredibly slow to build on my machine. I feel like moving to Vite would help a lot.

**Comment by mogeko** [comment]
It seems that Next.js has added [support for Rspack](https://github.com/vercel/next.js/tree/canary/packages/next-plugin-rspack) (very early on)

But I still hope that Next.js is decoupled from the build system, so that I can even migrate to platforms like Deno.
